modules/navier_stokes/src/kernels/INSBase.C

```diff
@@ -143,7 +145,7 @@ INSBase::strongViscousTermLaplace()
 RealVectorValue
```

Comment:
in the routine above, are we missing a grad_u * grad_mu term here?

from div(mu grad_u) = mu div grad u + grad_u . grad_mu

Reply:
I guess this isnt specific to RZ. Porbably a common approximation considering mu doesnt move much. Could be worth documenting?

Reply:
yes that is also true that we have not necessarily paid more attention to this term elsewhere

Reply:
I've opened https://github.com/idaholab/moose/issues/26217